episodes/26-collaborating.md

```diff
@@ -57,6 +57,142 @@ you may also want to assign these issues to a member of your lesson development
+   but leave the others unchecked for now. 
+
+
+### Submitting and Reviewing Pull Requests
+
+The Trainers will collaborate on a demonstration of the process laid out below.
+
+
+#### Pull Request Activity
+
+_Note: This activity is designed to be carried out by two Trainers acting in two different roles:
+"(S)" denotes the Pull Request Submitter role, "(R)" the Pull Request Reviewer._
+
+1. \(S) Explore list of issues on repository, choose issue to fix, make note of issue number.
+1. \(S) Edit a file to fix the issue, commit changes to a new branch with an appropriate name.
+        Ensure that the changes include some typo etc that the Reviewer
+        will be able to comment on and/or suggest a fix/improvement to.
+1. \(S) Show how to open a pull request from that new branch to `main`.
+1. \(S) In Pull Request description, reference the number of the issue being fixed, 
+        using a keyword ("Fixes", "Resolves") to ensure automatic closure.
+1. \(S) Demonstrate how to mark the PR as a draft, 
+        and then how to mark as it as ready for review.
+1. \(S) Request a review from the Pull Request Reviewer.
+1. \(R) Open the new Pull Request in GitHub.
+1. \(R) Explore the diff of the PR, 
+        by opening the _Files changed_ tab and talking through the display.
+1. \(R) Comment on a line in the diff.
+1. \(R) Suggest a change to the diff, 
+        using the "Add suggestion" button or three backticks marked `suggestion`.
+1. \(R) Complete your review by adding some summary text and selecting "Request changes".
+1. \(R) Return to _Conversation_ tab, 
+        to show how the PR thread looks after the review has been submitted.
+1. \(S) Navigate to PR and the _Files changed_ tab.
+1. \(S) Respond to Reviewer's comment(s), accept their suggested changes
+        (draw attention to the option to batch multiple suggestions into a single commit).
+1. \(S) Return to the _Conversation_ tab and request a new review from the Reviewer.
+1. \(R) Look again at the PR diff (_Files changed_) 
+        and ensure that requested changes have been made.
+1. \(R) Approve the PR.
+1. \(R) Thank the Submitter for their contribution.
+1. \(R) Merge the PR.
+1. \(R) Navigate to issue listing to confirm that the relevant issue has been 
+        automatically closed.
+
+
+:::::: challenge
+
+### Practice with Pull Requests (15 minutes)
+
+In breakout groups, assign yourselves the role of Submitter (S) and Reviewer (R),
+and repeat the process described above by completing the steps below.
+If anythere are any issues open on the repository that can be solved very quickly,
+feel free to address those with this exercise.
+Otherwise, the Submitter can open a Pull Request to make simple changes:
+typo fixes, or some completely arbitrary changes to a file.
+If the changes made for the exercise are not improvements to the lesson,
+the Reviewer should follow all the steps but close the PR at the end instead of merging it.
```

Comment:
Not sure what we mean here - are we demonstrating a case when the PR is not accepted?